README.md

```diff
@@ -42,10 +42,11 @@ This is the fork of the [original rounded-window-corners extension][14] by
 If you want to install the latest commit of the extension, you can get a
 pre-built archive from GitHub Actions.
 
-1. Go to [the build action page](https://github.com/flexagoon/rounded-window-corners/actions/workflows/build.yml)
-2. Click on the latest workflow run
-3. Download the extension from the "artifacts" section at the bottom
-4. Install it with the `gnome-extensions install` command
+1. Sign into GitHub.
```

Comment:
```suggestion
1. Sign in to GitHub.
```

https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to